eZ/Publish/API/Repository/Tests/ContentServiceTest.php

```diff
@@ -4819,4 +4819,33 @@ private function createFieldsFixture()
             ),
         );
     }
+
+    /**
+     * Test for the isContentTypeUsed() method.
+     *
+     * @see \eZ\Publish\API\Repository\ContentService::isContentTypeUsed()
+     * @group user
+     */
+    public function testIsContentTypeUsed()
+    {
+        if ($this->isVersion4()) {
+            $this->markTestSkipped('This test requires eZ Publish 5');
```

Comment:
@crevillo @bdunogier Since the test doesn't exist in pre-6 I guess I should just remove this if-clause.